diff -bur sumo_orig_test_2/src/traci-server/TraCIServerAPI_Vehicle.cpp sumo_orig_patch_test/src/traci-server/TraCIServerAPI_Vehicle.cpp
--- sumo_orig_test_2/src/traci-server/TraCIServerAPI_Vehicle.cpp	2017-10-12 22:22:35.000000000 -0700
+++ sumo_orig_patch_test/src/traci-server/TraCIServerAPI_Vehicle.cpp	2017-10-12 19:15:46.000000000 -0700
@@ -1103,7 +1103,8 @@
                     return server.writeErrorStatusCmd(CMD_SET_VEHICLE_VARIABLE, error, outputStorage);
                 }
                 if (vehicleParams.departProcedure == DEPART_GIVEN && vehicleParams.depart < MSNet::getInstance()->getCurrentTimeStep()) {
-                    return server.writeErrorStatusCmd(CMD_SET_VEHICLE_VARIABLE, "Departure time in the past.", outputStorage);
+                    vehicleParams.depart = MSNet::getInstance()->getCurrentTimeStep();
+                    // return server.writeErrorStatusCmd(CMD_SET_VEHICLE_VARIABLE, "Departure time in the past.", outputStorage);
                 }
 
                 if (!server.readTypeCheckingString(inputStorage, helper)) {
diff -bur sumo_orig_test_2/tools/build/version.py sumo_orig_patch_test/tools/build/version.py
--- sumo_orig_test_2/tools/build/version.py	2017-10-12 22:22:48.000000000 -0700
+++ sumo_orig_patch_test/tools/build/version.py	2017-10-13 04:27:43.000000000 -0700
@@ -124,7 +124,7 @@
             # svnFile is newer. lets update the revision number
             try:
                 svnInfo = subprocess.check_output(['svn', 'info', sumoSrc])
-                svnRevision = int(re.search('Revision: (\d*)', svnInfo).group(1))
+                svnRevision = int(re.search('Revision: (\d*)', repr(svnInfo)).group(1))
             except:
                 svnRevision = parseRevision(svnFile)
             create_version_file(versionFile, svnRevision, svnFile)
