diff -Naur sumo-svn/src/traci-server/TraCIServerAPI_Vehicle.cpp sumo-svn-patched/src/traci-server/TraCIServerAPI_Vehicle.cpp
--- sumo-svn/src/traci-server/TraCIServerAPI_Vehicle.cpp	2017-04-06 14:31:23.000000000 -0700
+++ sumo-svn-patched/src/traci-server/TraCIServerAPI_Vehicle.cpp	2017-04-11 13:54:16.000000000 -0700
@@ -1,3 +1,4 @@
+                vehicleParams.depart = MSNet::getInstance()->getCurrentTimeStep();
 /****************************************************************************/
 /// @file    TraCIServerAPI_Vehicle.cpp
 /// @author  Daniel Krajzewicz
@@ -1214,7 +1215,8 @@
                 return server.writeErrorStatusCmd(CMD_SET_VEHICLE_VARIABLE, error, outputStorage);
             }
             if (vehicleParams.departProcedure == DEPART_GIVEN && vehicleParams.depart < MSNet::getInstance()->getCurrentTimeStep()) {
-                return server.writeErrorStatusCmd(CMD_SET_VEHICLE_VARIABLE, "Departure time in the past.", outputStorage);
+                vehicleParams.depart = MSNet::getInstance()->getCurrentTimeStep();                
+                // return server.writeErrorStatusCmd(CMD_SET_VEHICLE_VARIABLE, "Departure time in the past.", outputStorage);
             }
 
             if (!server.readTypeCheckingString(inputStorage, helper)) {
